pulumitest/grpcLog.go

```diff
@@ -23,3 +27,15 @@ func (pt *PulumiTest) GrpcLog() *grpclog.GrpcLog {
 	}
 	return log
 }
+
+// ClearGrpcLog clears the gRPC log for the current stack based on the PULUMI_DEBUG_GRPC env var.
+func (pt *PulumiTest) ClearGrpcLog() {
+	env := pt.CurrentStack().Workspace().GetEnvVars()
+	if env == nil || env["PULUMI_DEBUG_GRPC"] == "" {
+		pt.t.Log("can't clear gRPC log: PULUMI_DEBUG_GRPC env var not set")
```

Comment:
Too chatty? Perhaps the default situation is exactly that PULUMI_DEBUG_GRPC is not set?